libs/static/src/types.ts

```diff
@@ -0,0 +1,28 @@
+export type Prefix = 'bsx' | 'glmr' | 'ksm' | 'movr' | 'rmrk2' | 'snek'
+// | 'statemine'
+// | 'westmint'
+
+export type BackwardPrefix = 'rmrk' & Prefix
+
+export type Squid = 'rubick' | 'snekk' | 'click' | 'antick'
```

Comment:
Maybe leave rmrk as is and add use ksm for rmrk2

Reply:
CC @Jarsen136

Reply:
the logical thing is that `rmrk` prefix is deeply written into code. Where as `rmrk2` is new prefix (easier to fix this rn). 


My next steps will be merging rmrk & rmrk2 into one indexer

Reply:
> this will be complete mess no?

No cause we dont have many occurences of rmrk2 in our codebase. 

As we spoke with you 3 weeks ago we want to replace rmrk mentions to kusama. 

So either best time is to hack it now or never

Reply:
also @yangwao 

code is currently really hacky 

```ts
It's a bit hard to read.

prefix === 'ksm' ? 'rmrk' : prefix,
prefix === 'rmrk' ? 'ksm' : prefix
Could we manage to unify them?
```

Reply:
and why we can't keep `rmrk2` @vikiival  ?